docs/docs/overview/quickstart/get-started-with-React.md

```diff
@@ -0,0 +1,230 @@
+---
+sidebar_position: 3
+sidebar_label: Get started with React
+---
+
+# React Quickstart
+
+Learn how to integrate Novu into your React.js app on the fly. Send notifications across channels (SMS, Email, Chat, Push) with the help of a rich, customizable, real-time In-App notification center
+
+## Requirements
+
+To follow the steps in this quickstart, you'll need:
+
+- A Novu account. [Sign up for free](http://web.novu.co) if you don’t have one yet.
+- A working React development environment.
+
+You can find the code for this project [here](https://github.com/sumitsaurabh927/react-fe-for-quickstart)
+
+## Create a new React app
+
+If you already have a working React app, please feel free to skip this section.
+
+Create a new React app by running the following command in your terminal:
+
+```bash
+npx create-react-app notifications-app
+```
+
+Wait a couple of minutes for all the packages and dependencies to get installed. Then `cd` into the `notifications-app` folder and now let's install the notification centre component.
+
+## Install Novu React Notification Center Package
+
+The Novu Vue package provides a React component library that you can use to add a fully functioning notification center into your React application.
```

Comment:
```suggestion
The Novu React package provides a React component library that you can use to add a fully functioning notification center into your React application.
```

Reply:
Thanks for pointing this out! 🙏

I've fixed it in the latest commit - 781d0dc